### PR TITLE
Issue 175 make sync chap 2commits

### PR DESCRIPTION
**Pull request by dsolt**
This introduces a new synchronization chapter in anticipation of a more thorough chapter explaining put/get.  That work will leave that chapter very full and the synchronization API, while short, doesn't really fit into that new chapter.

### CODE REVIEW

**Comment by dsolt** [comment]
When viewing this PR, I suggest you click on the individual commits (well, really the 2nd commit) to see what was changed.  Otherwise, it tries to show you the Chap_API_Sync as a new chapter and you can't see as easily what we changed.

**Comment by dsolt** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

    You do not need to vote on every proposal
    If you have no opinion, don't vote - that is also useful data
    If you've already commented on this issue, please still vote so
    we know your current thoughts
    Not all proposals solve exactly the same problem, so we may end
    up accepting proposals that appear to have some overlap
    This is not a binding majority-rule vote, but it will be a very
    significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

    Hooray or Rocket: I support this so strongly that I
    want to be an advocate for it
    Heart: I think this is an ideal solution
    Thumbs up: I'd be happy with this solution
    Confused: I'd rather we not do this, but I can tolerate it
    Thumbs down: I'd be actively unhappy, and may even consider
    other technologies instead
    If you want to explain in more detail, feel free to add another
    comment, but please also vote on this comment.

**Comment by dsolt** [comment]
[pmix-standard-sync.pdf](https://github.com/pmix/pmix-standard/files/8017351/pmix-standard-sync.pdf)

Here is a pdf showing the changes to this chapter with green highlighting for new text, red for changed text and grey for moved text.

File: Chap_API_Sync.tex [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+to synchronize a set of processes.  The fence operation can be useful after an application
+performs a number of \refapi{PMIx_Put} operations to coordinate with other processes that the
+data is available for access.   This avoids unsuccessful \refapi{PMIx_Get} calls that might
+otherwise be invoked before the cooresponding \refapi{PMIx_Put} call is complete.
+
+In its default form, the fence operation acts as a barrier between the processes and does not exchange data.
+Clients can pass the \refattr{PMIX_COLLECT_DATA} attribute to request 
+that the \refapi{PMIx_Fence} and \refapi{PMIx_Fence_nb} functions exchange all committed 
+data between all involved servers during the synchronization operation.
+This will make local to each remote process the data put by other processes resulting
```
Comment (abouteiller) [inline Chap_API_Sync.tex:28]: remove `remote`

**Comment by jjhursey** [comment]
PMIx ASC 1Q 2022

 * Passed the first vote: 11 yes / 0 no / 0 abstain
 * This is eligible for a second vote in the 2Q 2022 ASC quarterly meeting

**Comment by jjhursey** [comment]
PMIx ASC 2Q 2022
 * Passed the second vote: 11 yes / 0 no / 0 abstain
 * Will be merged into the next release

**Comment by jjhursey** [comment]
Rebased version in https://github.com/pmix/pmix-standard/pull/408

**Comment by jjhursey** [comment]
Closing in favor of PR #408 - which is the same content, but rebased onto the current `master`

**Pull request closed without merging**